orders/migrations/0003_static_orders.py

```diff
@@ -0,0 +1,154 @@
+                                 end_date=timezone.datetime(2022, 2, 8) + datetime.timedelta(days=10)),
+            Order.objects.create(daycare_id=all_daycares.get(pk=20), dog_owner_id=all_dog_owners.get(pk=20),
+                                 price_per_day=100, start_date=timezone.datetime(2022, 3, 11),
+                                 end_date=timezone.datetime(2022, 3, 11) + datetime.timedelta(days=6)),
+            Order.objects.create(daycare_id=all_daycares.get(pk=20), dog_owner_id=all_dog_owners.get(pk=21),
+                                 price_per_day=200, start_date=timezone.datetime(2022, 1, 17),
+                                 end_date=timezone.datetime(2022, 1, 17) + datetime.timedelta(days=4)),
+            Order.objects.create(daycare_id=all_daycares.get(pk=21), dog_owner_id=all_dog_owners.get(pk=21),
+                                 price_per_day=100, start_date=timezone.datetime(2022, 8, 23),
+                                 end_date=timezone.datetime(2022, 8, 23) + datetime.timedelta(days=7)),
+            Order.objects.create(daycare_id=all_daycares.get(pk=21), dog_owner_id=all_dog_owners.get(pk=22),
+                                 price_per_day=200, start_date=timezone.datetime(2022, 3, 12),
+                                 end_date=timezone.datetime(2022, 3, 12) + datetime.timedelta(days=2)),
+            Order.objects.create(daycare_id=all_daycares.get(pk=22), dog_owner_id=all_dog_owners.get(pk=22),
+                                 price_per_day=100, start_date=timezone.datetime(2022, 3, 5),
+                                 end_date=timezone.datetime(2022, 3, 5) + datetime.timedelta(days=7)),
+            Order.objects.create(daycare_id=all_daycares.get(pk=22), dog_owner_id=all_dog_owners.get(pk=23),
+                                 price_per_day=200, start_date=timezone.datetime(2022, 4, 4),
+                                 end_date=timezone.datetime(2022, 4, 4) + datetime.timedelta(days=3)),
+            Order.objects.create(daycare_id=all_daycares.get(pk=23), dog_owner_id=all_dog_owners.get(pk=23),
+                                 price_per_day=100, start_date=timezone.datetime(2022, 5, 15),
+                                 end_date=timezone.datetime(2022, 5, 15) + datetime.timedelta(days=6)),
+            Order.objects.create(daycare_id=all_daycares.get(pk=23), dog_owner_id=all_dog_owners.get(pk=4),
+                                 price_per_day=200, start_date=timezone.datetime(2022, 5, 5),
+                                 end_date=timezone.datetime(2022, 5, 5) + datetime.timedelta(days=8))
+        ]
+
+        approved_orders_indexes = [1, 4, 7, 10, 14, 16, 17, 20, 24, 27, 30, 32, 35, 36, 39]
+        canceled_orders_indexes = [2, 3, 12, 18, 22, 29, 31, 38]
+        with transaction.atomic():
+            for i in approved_orders_indexes:
+                static_orders_list[i].approve_order()
+            for i in canceled_orders_indexes:
```

Comment:
is all this needed? why don't you create them as approved/canceled order?

Reply:
In the "create" method on the "Order" model, when an order is created, it's always on "Pending" status, because the dog owner is waiting for the daycare to approve/decline the order. So after creating an order, the "approve_order()"/ "cancel_order()" methods must be called.
I can call the methods right after the creation of each of the orders for better readability. 
Let me know what you think.

Reply:
Please change it in away that there is no need to keep the indexes of each type of `order`.
I would be nice if there were helpers for creating approved /  canceled orders.

Reply:
When creating the static data you don't need to follow how it'll work in real-time, you just insert the data the way you want it to be. if you are doing more operations on it, it is not that "static" anymore. 
You can create the objects as approved/cancelled, no need for extra steps.

Reply:
I removed the indexes for each order's approval/cancellation, and add to order's "create" method a default parameter of "StatusOptions.Pending".

BUT, for better readability, and explicit creation of a static order, with the desired status, I've created orders with "StatusOptions.Pending" status on the migration file.

Reply:
much MUCH better! good job